tests/Integration/ResourceStatusSystem/Test_ResourceStatus.py

```diff
@@ -128,7 +129,7 @@ def test_addAndRemove(self):
     self.assertTrue(res['OK'])
     self.assertEqual(res['Value'][0][0], 'TestName1234')
     self.assertEqual(res['Value'][0][1], 'statusType')
-    self.assertEqual(res['Value'][0][2], 'Banned')
+    self.assertEqual(res['Value'][0][3], 'Banned')
     print("inserted lastCheckTime and DateEffective: %s, %s" % (res['Value'][0][7], res['Value'][0][4]))
```

Comment:
you also need to increment these indices (7 and 4), also in other places.

Reply:
Thank you for spotting this. Done.